source/fundamentals/enterprise-auth.txt

```diff
@@ -280,3 +287,150 @@ mechanism:
       method. The code to instantiate a ``MongoClient`` resembles the following:
 
       .. include:: /includes/fundamentals/code-snippets/auth-credentials-ldap.rst
+
+.. _mongodb-oidc:
+
+``MONGODB-OIDC``
+~~~~~~~~~~~~~~~~
+
+.. important::
+
+   The MONGODB-OIDC authentication mechanism requires MongoDB v7.0 or later running
```

Comment:
S: I'd use {+mdb-server+} here
```suggestion
   The MONGODB-OIDC authentication mechanism requires {+mdb-server+} v7.0 or later running
```

Reply:
good thought. this content was taken from pymongo, which doesn't have that constant.